llvm/lib/Transforms/Instrumentation/CMakeLists.txt

```diff
@@ -43,3 +43,5 @@ add_llvm_component_library(LLVMInstrumentation
   TransformUtils
   ProfileData
   )
+
+  set_property(TARGET LLVMInstrumentation APPEND_STRING PROPERTY COMPILE_FLAGS " -g -O0")
```

Comment:
nit: missing EOL

Reply:
sorry, this is debug change is not intended for commit